pymc/model_graph.py

```diff
@@ -430,6 +430,9 @@ def edges(
         ]
 
 
+GraphAttrMapping = dict[Any, Any]
```

Comment:
```suggestion
GraphAttrMapping = dict[str, Any]
```

I'm just guessing, but this is probably more correct?